lib/matplotlib/colorbar.py

```diff
@@ -817,26 +816,30 @@ def _get_ticker_locator_formatter(self):
         _log.debug('locator: %r', locator)
 
     @_api.delete_parameter("3.5", "update_ticks")
-    def set_ticks(self, ticks, update_ticks=True):
+    def set_ticks(self, ticks, update_ticks=True, labels=None, *,
+                  minor=False, **kwargs):
         """
         Set tick locations.
 
         Parameters
         ----------
-        ticks : array-like or `~matplotlib.ticker.Locator` or None
-            The tick positions can be hard-coded by an array of values; or
-            they can be defined by a `.Locator`. Setting to *None* reverts
```

Comment:
Accepting a `Locator` is (intentionally?) no longer documented.

Reply:
Right, I didn't want to break API, however, `axis.set_ticks` does not take a locator so far as I can tell.